lib/k8s/resource_client/exec.rb

```diff
@@ -63,7 +63,11 @@ def exec(name:, namespace: @namespace, command:, container:, stdin: true, stdout
             ws = @transport.build_ws_conn(exec_path, query)
 
             ws.on :message do |event|
-              out = event.data.pack("C*")
+              # Previously we were packing the event.data into bytes, i.e.
```

Comment:
Should we have a check here since it's odd that it `suddenly` changed format?

Reply:
Good idea...